modules/local/report.nf

```diff
@@ -364,15 +366,24 @@ def create_action_call(sample_data){
                 qual_message.add("[PASSED] All Checks passed")
                 sample_status = "PASSED"
             }
-            qual_message.add("Passed Tests: ${checks - checks_failed - checks_ignored}/${checks}")
 
+            def organism_criteria = sample_data[val.key][species_tag]
+            def tests_passed = "Passed Tests: ${checks - checks_failed - checks_ignored}/${checks}"
+            qual_message.add(tests_passed)
 
-            qual_message.add("Species ID: ${val.value[val.key][params.top_hit_species.report_tag]}")
+            def species_id = "Species ID: ${val.value[val.key][params.top_hit_species.report_tag]}"
+            qual_message.add(species_id)
 
             // Qual summary not final message
             final_message = qual_message.join("\n")
             def terminal_message = populate_qual_message(qual_data).join("\n")
             log.info "\n$val.key\n${terminal_message}\n${sample_status}\n${final_message}"
+
+            // Reseq recommended should go to a seperate field
+            // Requested output should be: [PASS|FAILED] Species ID: [species] [Tests passed] [Organism criteria available]
+            qc_message = "${sample_status} ${species_id} ${tests_passed} Organism QC Criteria: ${organism_criteria}"
+
+            sample_data[val.key]["QCMessage"] = qc_message
```

Comment:
I am wondering if we should swap the role of `QCMessage` and `QCSummary` here? That is, `QCMessage` contains the original message and `QCSummary` is modified to contain the shortened QC messages. That way, the metadata field `QC Summary` doesn't have to change.